doc/source/reference/c-api/array.rst

```diff
@@ -750,13 +750,13 @@ General check of Python Type
 
     Evaluates true if *op* is either a Python scalar object (see
     :c:func:`PyArray_IsPythonScalar`) or an array scalar (an instance of a sub-
-    type of :c:data:`PyGenericArr_Type` ).
+    type of ``PyGenericArr_Type`` ).
```

Comment:
Another option would be to add `PyGenericArr_Type` to the `reference/c-api/types-and-structures.rst` page, just before `PyArrayDescr_Type and PyArray_Descr`. But maybe not worth the effort.

Reply:
I added it in the new push.